MovieQuiz/Services/QuestionFactory.swift

```diff
@@ -83,36 +83,37 @@ class QuestionFactory: QuestionFactoryProtocol {
            
             do {
                 imageData = try Data(contentsOf: movie.resizedImageURL)
+            
+                let rating = Float(movie.rating) ?? 0
+                
+                
+                let (text, correctAnswer) = self.generateTextAnswer(rating: rating)
+                
+                let question = QuizQuestion(image: imageData,
+                                             text: text,
+                                             correctAnswer: correctAnswer)
+                
+                DispatchQueue.main.async { [weak self] in
+                    guard let self else { return }
+                    self.delegate?.didReceiveNextQuestion(question: question)
+                }
+
             } catch {
                 print("Failed to load image")
                 //возвращаемся в главный поток, сетевые данные не удалось получить, работа с ними окончена
                 DispatchQueue.main.async { [weak self] in
-                    guard let self = self else { return }
+                    guard let self else { return }
                     self.delegate?.didFailToLoadImage()
                 }
             }
-            
-            let rating = Float(movie.rating) ?? 0
-            
-            
-            let (text, correctAnswer) = self.generateTextAnswer(rating: rating)
-            
-            let question = QuizQuestion(image: imageData,
-                                         text: text,
-                                         correctAnswer: correctAnswer)
-            
-            DispatchQueue.main.async { [weak self] in
-                guard let self = self else { return }
-                self.delegate?.didReceiveNextQuestion(question: question)
-            }
         }
     }
     
-    private func generateTextAnswer(rating: Float) -> (text: String, correctAnswer: Bool){
+    private func generateTextAnswer(rating: Float) -> (text: String, correctAnswer: Bool) {
         let number = Int.random(in: 1..<10)
         let word = ["больше", "меньше"].randomElement()
         let text: String
-        if let word = ["больше", "меньше"].randomElement() {
+        if let word = word {
```

Comment:
супер, а теперь можно использовать короткий синтаксис: 
```suggestion
        if let word {
```